content/filterx/function-reference.md

```diff
@@ -0,0 +1,406 @@
+---
+title: Filterx function reference
+linkTitle: Functions
+weight: 2500
+---
+
+<!-- This file is under the copyright of Axoflow, and licensed under Apache License 2.0, except for using the Axoflow and AxoSyslog trademarks. -->
+
+{{< include-headless "chunk/filterx-experimental-banner.md" >}}
+
+This page describes the functions you can use in [filterx blocks]({{< relref "/filterx/_index.md" >}}).
+
```

Comment:
We might want to explain the signatures a bit.

Arguments in a function can be either mandatory or optional.
* Mandatory options are always positional, so you need to pass them in the correct order. They cannot be set in `arg=value` format.
* Optional arguments are always named, like `arg=value`, and you can omit it or pass it in any order you want.